practice-app/back-end/routes/exchangerates.js

```diff
@@ -0,0 +1,16 @@
+var express = require('express');
+var router = express.Router();
+const axios = require('axios');
+
+router.get('/', (req, res, next) => {
+  axios.get('https://api.exchangeratesapi.io/latest?base=USD')
+  .then(response => {
+    console.log(res.json(response.data))
+    res.send(response.data)
```

Comment:
For our project, it would be more appropriate to convert the currency to TRY and console.log(...) should be deleted.